pkg/distro/fedora/images.go

```diff
@@ -687,3 +688,13 @@ func makeOSTreePayloadCommit(options *ostree.ImageOptions, defaultRef string) (o
 		RHSM: options.RHSM,
 	}, nil
 }
+
+// initialSetupKickstart returns the File configuration for a kickstart file
+// that's required to enable initial-setup to run on first boot.
+func initialSetupKickstart() *fsnode.File {
+	file, err := fsnode.NewFile("/root/anaconda-ks.cfg", nil, "root", "root", []byte("firstboot --reconfig\n"))
```

Comment:
@achilleas-k could you please add the comment to this file that it was created by osbuild? Or something along those lines...

I won't comment on each distro file, but ideally please add it to each of them.

Reply:
Added the following comment to the first two lines of all `anaconda-ks.cfg` files:
```
# Run initial-setup on first boot
# Created by osbuild
```